src/components/navbar/index.js

```diff
@@ -0,0 +1,101 @@
+import './_navbar.scss';
+import React from 'react';
+import {connect} from 'react-redux';
+import {Redirect, Link} from 'react-router-dom';
+
+import Icon from '../icon-component';
+import Avatar from '../avatar';
```

Comment:
We can take this out because we aren't planning to incorporate avatars at the moment.